### PR TITLE
Feature/uweb 5177 tabbed sequence

### PR DESCRIPTION
**Pull request by zwaardje**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -44,11 +44,6 @@ podTemplate(
               sh 'npm run lint'
             }
           },
-          'unit test': {
```
Comment (dan-entous) [inline Jenkinsfile]: does this need to be re-instated?

File: Jenkinsfile [hunk 2]
```diff
@@ -44,11 +44,6 @@ podTemplate(
               sh 'npm run lint'
             }
           },
```
Comment (dan-entous) [inline Jenkinsfile]: `removal of 'unit test'`
does this need to be re-instated or are we going to wait for the browserstack replacement?

File: src/js/components/organisms-page/content-cover/content-cover.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<article class="content-cover content-cover--cover">
+    <c-image v-if="this.image" v-bind="this.image"></c-image>
+
+    <div class="content-cover__image--overlay"></div>
+
+    <section class="content-cover__container">
+        <div class="flexgrid__cell--xs-6 content-cover-content content-cover--cover-reset">
```
Comment (dan-entous) [inline src/js/components/organisms-page/content-cover/content-cover.html]: `h3 indentation`

File: src/js/components/organisms-page/content-cover/content-cover.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+             <h3 class="content-cover__title">{{ title }}</h3>
```
Comment (dan-entous) [inline src/js/components/organisms-page/content-cover/content-cover.html]: `v-if description`
description is indicated as required in the component. why is `v-if` used here?

File: src/js/components/organisms-page/content-cover/content-cover.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        <p class="content-cover__description" v-if="description">{{ description }}</p>
+        <ContentButton v-bind="cta"
+                   class="content-cover__button button"
```
Comment (dan-entous) [inline src/js/components/organisms-page/content-cover/content-cover.html]: `v-if="cta ... & cta`
why is cta mentioned twice in the v-if statement?

File: src/js/components/organisms-page/content-cover/content-cover.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<article class="content-cover content-cover--cover">
+    <c-image v-if="this.image" v-bind="this.image"></c-image>
+
+    <div class="content-cover__image--overlay"></div>
+
+    <section class="content-cover__container">
+        <div class="flexgrid__cell--xs-6 content-cover-content content-cover--cover-reset">
+             <h3 class="content-cover__title">{{ title }}</h3>
+        <p class="content-cover__description" v-if="description">{{ description }}</p>
```
Comment (dan-entous) [inline src/js/components/organisms-page/content-cover/content-cover.html]: `multiple components & v-if="cta ... & cta`
* why is cta mentioned twice in the v-if statement?
* same question for the other components below
* could this be a dynamic component using something like `<component v-if="cta" :is="cta.type" :class="getClasses(cta.type)"/>`?

File: src/js/components/organisms-page/content-cover/content-cover.ts [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
```
Comment (dan-entous) [inline src/js/components/organisms-page/content-cover/content-cover.ts]: `extra line break`

File: src/stories/organisms/content-cover/content-cover.stories.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+storiesOf('organisms|content', module)
+    .addDecorator(withKnobs)
+    .add('content cover', () => ({
+        props: {
+            contentCover: {default: knobs().contentCover}
+        },
+        template: require('./content-cover.stories.html')
+    }),
+    {
```
Comment (dan-entous) [inline src/stories/organisms/content-cover/content-cover.stories.js]: `notes: markdown`
* storybook changed this format to: `notes: require('./content-cover.stories.md')`

File: src/stories/organisms/content-cover/content-cover.stories.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
+# Hero Cover
```
Comment (dan-entous) [inline src/stories/organisms/content-cover/content-cover.stories.md]: `title`
Hero Cover vs Content Cover

File: Jenkinsfile (hunk 2 above)
Comment (zwaardje) [reply Jenkinsfile]: Fixed on rebase

File: src/js/components/organisms-page/content-cover/content-cover.html (hunk 6 above)
Comment (zwaardje) [reply src/js/components/organisms-page/content-cover/content-cover.html]: It should be solved better, I agree. But for the sake of speed i'm now adding it like this. We should create a proper component for this.

File: Jenkinsfile [hunk 10]
```diff
@@ -47,11 +47,14 @@ podTemplate(
...
+<<<<<<< HEAD
```
Comment (dan-entous) [inline Jenkinsfile]: `<<<<<<< HEAD`

File: dist/components.css [hunk 11]
```diff
@@ -104,7 +104,11 @@ because both families serve another purpose. Therefore the line-heights will be
   margin: 0 0 6px 0;
   padding: 0; }
 
+<<<<<<< HEAD
```
Comment (dan-entous) [inline dist/components.css]: `<<<<<<< HEAD`
looks like the rebase conflict is still in the file

File: dist/components.js [hunk 12]
```diff
@@ -2971,11 +2971,19 @@ var _components = __webpack_require__(114);
```
Comment (dan-entous) [inline dist/components.js]: `<<<<<<< HEAD`
looks like the rebase conflict is still in the file

File: package.json [hunk 13]
```diff
@@ -45,7 +45,11 @@
         "vuex": "^3.1.0"
     },
     "devDependencies": {
+<<<<<<< HEAD
```
Comment (dan-entous) [inline package.json]: `<<<<<<< HEAD`
looks like the rebase conflict is still in the file

File: src/js/components/organisms-page/content-cover/content-cover.html (hunk 6 above)
Comment (dan-entous) [reply src/js/components/organisms-page/content-cover/content-cover.html]: added the change to branch

**Pull request merged**